core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3378,16 +3389,6 @@ public boolean recordActiveSyncTxid(long txId, long mountId) {
     return true;
   }
 
-  private boolean syncMetadata(RpcContext rpcContext, AlluxioURI path,
```

Comment:
this overloading method is not helping much. Consolidating two `syncMetadata` methods so it is very clear to see all methods exercise this code path, easier to reason

Reply:
Do we still plan to do this?